packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/ai-agent-action/utils/formatChatMessageString.ts

```diff
@@ -0,0 +1,13 @@
+import { DateTime } from 'luxon';
+
+export const formatChatMessageDate = (date: string) => {
```

Comment:
Check date-utils.ts if there's something you can reuse or improve, this shouldn't be an isolated util as it's a generic use-case